### ISSUE TITLE
Tooltip Variation - Glossary Entry

### ISSUE THREAD

**Issue by emsaiz** [OER-WEB-TEAM/drupal-subtheme--nihod#64]
![image](https://github.com/OER-WEB-TEAM/design-system--drupal-theme/assets/59488566/3af748ab-3931-4885-909a-bba57d793fa8)

**Comment by emsaiz** [comment]
Glossary Entry variation on link page would look like: 
https://xd.adobe.com/view/124158b0-1037-4ee2-b081-aef1353d537a-4712/
![image](https://github.com/OER-WEB-TEAM/design-system--drupal-theme/assets/59488566/0bea4864-b982-4297-a80f-c3ff31981ac9)

**Comment by emsaiz** [comment]
**Glossary Entry variation on link page would look like:** 

![image](https://github.com/OER-WEB-TEAM/design-system--drupal-theme/assets/59488566/4d6606fd-4b1c-435b-8409-3779a3377763)

**Comment by AlexVanK** [comment]
This is not a variation of a link, it's a variation of the tooltip. Since a tooltip can be triggered by anything.
Otherwise by this logic, you could have modals as a variation of links because they can be triggered by links :stuck_out_tongue_winking_eye:

---

Now the real question is, Glossary probably needs to be supported on a website other than just a tooltip.
Either a content-editor maintains a glossary on Drupal, or on a data-file (which I prefer because more independent).
Then tooltips are displayed where the glossary term pops up, but there should be also a page that lists all the terms together.

But if this is the case, I will have to work on such a system - easy to do, but maybe we need to share the ideas with David first?

**Comment by emsaiz** [comment]
Yeah I have it as a variation on both links and tooltips but I agree, tooltips makes more sense. IDK why I shared this mockup over the tooltip one. 

I know almost nothing about how our current glossary works. Are you saying you would need to build an additional content-editor or data-file to support the current glossary in order to make something like this work? Or would those things replace the current glossary?

**Comment by AlexVanK** [comment]
Not sure what current glossary you are referring to, but it's probably wrong! LOL
Yes, I'm referring to having to build a new system that is Drupal independent, so it can be easily managed, and used on all the sites... This is a high-level idea, I would have to think about how exactly implementing it :)

Probably other than the way it's displayed, beyond the scope of the Design Syst....
We can talk about it next week

**Comment by emsaiz** [comment]
Sounds good! 

just for reference: https://grants.nih.gov/grants/glossary.htm
